### DISCUSSION TITLE
Not able to connect MYSQL with Budibase

### DISCUSSION THREAD

**Discussion by akm802594** [Budibase/budibase#3133] [Help]
Dear Team,

I am trying to connect the My SQL database to Budibase. In doing so we are not able to do the connect.
Attached is the word documents with the screenshots.
Please help us to know where I am going wrong.
The SQL server details are :
User : Admin
Password: admin1234
DataBase : Spare_Management
[SCREENSHOTS (1).docx](https://github.com/Budibase/budibase/files/7407733/SCREENSHOTS.1.docx)

**Comment by shogunpurple** [comment]
Hi @akm802594,

I think that your `SERVER` option is incorrect. That does not look like a valid IP address or domain, unless you have a specific DNS entry that points to your database with that name.

Is your DB hosted on the same machine that budibase is? Or is it a shared DB server that you have hosted elsewhere?
